include/library/spdm_lib_config.h

```diff
@@ -311,7 +311,7 @@
 #define LIBSPDM_TRANSPORT_ADDITIONAL_SIZE    64
 #endif
 #ifndef LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE
-#define LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE (LIBSPDM_DATA_TRANSFER_SIZE + \
+#define LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE (LIBSPDM_MAX_MESSAGE_BUFFER_SIZE + \
                                             LIBSPDM_TRANSPORT_ADDITIONAL_SIZE)
 #endif
 
```

Comment:
Please add comment into the code, where you describe the SEND/RECEIVE buffer.

The rang of 0 ~ LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE is for Secure encoding/
decoding, LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE ~ End is for Chunk.

Reply:
Please add detail section info for LIBSPDM_SCRATCH_BUFFER_SIZE at line 348.